test/Confluent.Kafka.UnitTests/Confluent.Kafka.UnitTests.csproj

```diff
@@ -17,4 +17,8 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Forgot to remove it, It does just add it by itself in VS2017 when using test explorer (kind of guid) Looked for a way to tell the project is a unittest (like in the stack overflow link) but no luck so far on .netcoreapp